blocks/library/paragraph/index.js

```diff
@@ -254,6 +251,11 @@ export const settings = {
 		],
 	},
 
+	autocompleters: [
```

Comment:
It probably makes the most sense to have it here 👍 Let's see how it goes :)

Reply:
You might want different autocompleters per block. A concrete example is that a block autocompleter makes sense in a paragraph but not in the caption field of a cover image. If you offer block completion there, it is more likely an opportunity for surprising replacement than a feature IMO.

Reply:
What If you want different autocompleters per RichText? is this a valid use case? could this be a prop of `RichText` instead?

Reply:
I'm not sure, but it _seems_ reasonable to want different autocompleters per RichText. I'm not sure what overriding should look like in that case though.

I'm wondering whether this PR is trying to do too much and, in retrospect, would like to have created an autocompletion extensibility issue for discussion and smaller PRs. But if we can adjust in the future, maybe it would be good to continue as-is and get more feedback once merged.

Reply:
Sounds good to me @brandonpayton